src/components/card/Description.vue

```diff
@@ -238,6 +253,65 @@ export default {
 				await this.saveDescription()
 			}, 2500)
 		},
+		async addLastAttachmmentOnCursor() {
+			const attachementLength = this.$store.getters.attachmentsByCard(this.card.id).length
+			if (attachementLength === 0) return
+
+			const attachment = this.$store.getters.attachmentsByCard(this.card.id)[attachementLength - 1]
+			const descStringLines = this.$refs.markdownEditor.easymde.value().split('\n')
+			const cursor = this.lastPasteImageCursor === null ? this.$refs.markdownEditor.easymde.codemirror.getCursor() : this.lastPasteImageCursor
+			let embed = ''
+			if ((attachment.type === 'file' && attachment.extendedData.hasPreview) || attachment.extendedData.mimetype.includes('image')) {
+				embed = '!'
+			}
+
+			const attachmentString = embed + '[📎 ' + attachment.data + '](' + this.attachmentPreview(attachment) + ')'
+			let lineUpatedLength = 0
+			for (let i = 0; i < descStringLines.length; i++) {
+				if (i === cursor.line) {
+					descStringLines[i] = descStringLines[i].substring(0, cursor.ch) + attachmentString + descStringLines[i].substring(cursor.ch)
+					lineUpatedLength = descStringLines[i].length
+					break
+				}
+			}
+			const newContent = descStringLines.join('\n')
+			this.$refs.markdownEditor.easymde.value(newContent)
+			this.description = newContent
+			this.modalShow = false
+			this.updateDescription()
+
+			if (lineUpatedLength > 0) {
+				await this.$nextTick()
+				this.lastPasteImageCursor = { line: cursor.line, ch: lineUpatedLength }
+				this.$refs.markdownEditor.easymde.codemirror.focus()
+				this.$refs.markdownEditor.easymde.codemirror.setCursor({ line: cursor.line, ch: lineUpatedLength })
+			}
+
+		},
+		onPaste() {
+			if (!this.descriptionEditing) return
+
+			const clipboardData = event.clipboardData
+			let files = []
+			if (clipboardData.files && clipboardData.files.length > 0) {
+				files = clipboardData.files
+			} else if (clipboardData.items && clipboardData.items.length > 0) {
+				for (let i = 0; i < clipboardData.items.length; i++) {
+					if (clipboardData.items[i].kind === 'file') {
+						files.push(clipboardData.items[i].getAsFile())
+					}
+				}
+			}
+
+			if (files.length === 0) return
```

Comment:
```suggestion
			if (files.length === 0) {
				return
			}
```

To stick to https://docs.nextcloud.com/server/latest/developer_manual/getting_started/codingguidelines.html#id2 and makes the code a bit easier to grasp.